ha/setup-cluster.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -x
+
+WORK_DIR="${1:-$(pwd)}"
+CONFIG_DIR="${2:-"${WORK_DIR}/config"}"
+
+CREATE_VM_SCRIPT="$(pwd)/create-vm.sh"
+
+VM01="node01"
+VM02="node02"
+VM03="node03"
+
+SSH="ssh -o StrictHostKeyChecking=no"
+SCP="scp -o StrictHostKeyChecking=no"
+
```

Comment:
I'd add a `check_requirements()` function that checks that all the dependencies are installed. Can be as simple as (untested):
```
check_requirements() {
    hash cmd1 cmd2 cmd3 || exit 127
}
```